SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/Http/HttpNetworkAdapter.cs

```diff
@@ -125,6 +132,10 @@ private async Task<HttpResponseMessage> SendRequestAsync(HttpRequestMessage requ
             {
                 return await HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
             }
+            catch (HttpRequestException e)
+            {
+                throw new NetworkIngestionException(e);
```

Comment:
So how does 4XX vs 5XX work then?

Reply:
All server answers with any codes does not throw this exception (there is an API to do it, but not by default). Handling error response codes is not changed

Reply:
Do we know the exact conditions when this is thrown then?

Reply:
Yes, for example SSL errors.
The code for testing:
```C#
AppCenter.SetLogUrl("https://self-signed.badssl.com");
```

Reply:
From original issue it was `The SSL connection could not be established, see inner exception.`, probably timeout on handshake or so. Anyway, this type of exceptions is intended to be retried